.spec/4412-publish-closure/acceptance.md

```
@@ -0,0 +1,115 @@
+### ci-gate
+- [ ] `just ci-gate` includes: `just ci-publish-closure`
+- [ ] Step runs after `just hook-tests`
+- [ ] ci-gate passes on master
+
+## Test Criteria
+
+### Unit tests
+- [ ] `cargo test -p xtask -- publish_closure` passes
+- [ ] Test `publish_closure_passes_on_master` passes
+- [ ] Test `publish_closure_single_crate_flag` passes
+- [ ] Test `publish_closure_unknown_crate_exits_nonzero` passes
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)